hw1.txt

```diff
@@ -0,0 +1,32 @@
+--CREATE A TABLE
+create table person (
+	ID SERIAL UNIQUE,
+	NAME CHARACTER VARYING(30),

```

Comment:
It's better to use `VARCHAR` - it's more common

Reply:
Corrected